libdevcore/FixedHash.h

```diff
@@ -141,6 +141,12 @@ class FixedHash
 	/// @returns a constant byte pointer to the object's data.
 	byte const* data() const { return m_data.data(); }
 
+	/// @returns begin iterator.
+	auto begin() const -> typename std::array<byte, N>::const_iterator { return m_data.begin(); }
```

Comment:
I don't think some, `cbegin()` is only useful when you have a non-const object, with `begin() const` and `begin()` and you want to enforce getting const iterator.